glambie/processing/processing_helpers.py

```diff
@@ -132,8 +130,9 @@ def convert_datasets_to_annual_trends(data_catalogue: DataCatalogue,
     return catalogue_annual_grid
 
 
-def convert_datasets_to_longterm_trends(data_catalogue: DataCatalogue, year_type: YearType,
-                                        season_calibration_dataset: Timeseries) -> DataCatalogue:
+def convert_datasets_to_longterm_trends_in_unit_mwe(data_catalogue: DataCatalogue, year_type: YearType,
+                                                    season_calibration_dataset: Timeseries,
+                                                    min_max_time_window: Tuple[float, float] = None) -> DataCatalogue:
```

Comment:
A thought on a clearer name?
```suggestion
                                                    output_trend_date_range: Tuple[float, float] = None) -> DataCatalogue:
```